### PR TITLE
Added reclaim-mannequin to ado2gh

### PR DESCRIPTION
**Pull request by dylan-smith**
Issue #356 

- [x] Did you write/update appropriate tests
- [x] Release notes updated (if appropriate)
- [x] Appropriate logging output
- [x] Issue linked
- [x] Docs updated (or issue created) - #360

### CODE REVIEW

**Review by ArinGhazarian — CHANGES_REQUESTED** [review]

File: src/ado2gh/Commands/ReclaimMannequinCommand.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+            var githubApi = _githubApiFactory.Create();
+
+            _log.LogInformation($"GITHUB ORG: {githubOrg}");
+            var githubOrgId = await githubApi.GetOrganizationId(githubOrg);
+            _log.LogInformation($"    Organization Id: {githubOrgId}");
+
+            var mannequin = await githubApi.GetMannequin(githubOrgId, mannequinUser);
+
+            if (mannequin == null || mannequin.Id == null)
```
[the remark this answers is not in the document]
Comment (ArinGhazarian) [reply src/ado2gh/Commands/ReclaimMannequinCommand.cs:80]: you're right they're not semantically equivalent, my bad 😞

**Comment by dylan-smith** [comment]
Since we don't have the feature implemented yet to consume this csv, my suggestion is to make this command hidden until we add in the feature that can consume it to claim mannequins in bulk (IsHidden = true on the command).

File: src/ado2gh/GithubApiFactory.cs [hunk 2]
```diff
@@ -21,6 +21,11 @@ public GithubApiFactory(OctoLogger octoLogger, HttpClient client, EnvironmentVar
             _versionProvider = versionProvider;
         }
 
+        public virtual GithubApi Create()
+        {
+            return Create(null, null);
+        }
```
[the remark this answers is not in the document]
Comment (dylan-smith) [reply src/ado2gh/GithubApiFactory.cs]: I reverted this

File: src/ado2gh/Commands/ReclaimMannequinCommand.cs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                IsRequired = false
+            };
+
+            AddOption(githubOrgOption);
+            AddOption(mannequinUsernameOption);
+            AddOption(targetUsernameOption);
+            AddOption(forceOption);
+            AddOption(verbose);
+
+            Handler = CommandHandler.Create<string, string, string, bool, bool>(Invoke);
```
[the remark this answers is not in the document]
Comment (dylan-smith) [reply src/ado2gh/Commands/ReclaimMannequinCommand.cs]: done

File: src/ado2gh/Commands/ReclaimMannequinCommand.cs [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+            if (result.Data.CreateAttributionInvitation != null &&
+                result.Data.CreateAttributionInvitation.Source.Id == mannequin.Id &&
+                result.Data.CreateAttributionInvitation.Target.Id == targetUserId)
+            {
+                _log.LogInformation($"Successfully reclaimed {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId})");
+            }
+            else
+            {
+                throw new OctoshiftCliException($"Failed to reclaim {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId})");
+            }
```
[the remark this answers is not in the document]
Comment (dylan-smith) [reply src/ado2gh/Commands/ReclaimMannequinCommand.cs]: done, also done in the gei version of this code

**Review by ArinGhazarian — APPROVED** [review]
💯

**Pull request merged**